src/assets/config.json

```diff
@@ -64,5 +64,13 @@
 
   "swal": {
     "timer": 3000
+  },
+
+  "scoreboard": {
+    "room_name": null,
+    "team_name": null,
+
+    "scoreboardUrl": "scoreboard.html",
+    "scoreboardApiUrl": "http://scoreboard.test"

```

Comment:
This should just be `/scoreboard` right? Why is this a config setting?

Reply:
Since Lumen is installed in a subdirectory, this is not working for me personally. Is something that needs to be discussed.